store/postgres_migration_test.go

```diff
@@ -1,151 +1,147 @@
 package store
 
 import (
-	"database/sql"
+	"context"
 	"fmt"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
-	"github.com/valocode/bubbly/config"
 	"github.com/valocode/bubbly/env"
-
-	"github.com/ory/dockertest/v3"
-	"github.com/ory/dockertest/v3/docker"
-	"github.com/valocode/bubbly/api/core"
-
-	"github.com/stretchr/testify/require"
+	"github.com/valocode/bubbly/test"
 )
 
 const (
 	postgresDatabase = "bubbly"
 	postgresUser     = "postgres"
 	postgresPassword = "secret"
```

Comment:
❓ These probably are not necessary since they are duplicated in the `test/dockertest.go`?

Reply:
Well they are being used in `store/store_test.go`... This was a real PITA yesterday because the tests are coupled to each other. Again, `bCtx` as the source of truth would be great!